core/main/src/service/apps/delegated_launcher_handler.rs

```diff
@@ -288,7 +288,16 @@ pub struct DelegatedLauncherHandler {
 /*
 Tell lifecycle metrics which methods map to which metrics AppLifecycleStates
 */
-fn map_event(event: &AppMethod) -> Option<AppLifecycleState> {
+
+fn map_event(event: &AppMethod, inactive: bool) -> Option<AppLifecycleState> {
```

Comment:
I'm a little confused by what "inactive" means here, it doesn't just mean the app is in the ``LifecycleState::Inactive`` state does it?

Reply:
it means it was launched as inactive so a lifecycle.ready during a launch of inactive means its actually suspended. Thats the the current change to handle apps which are launched with an intent to be suspended. Metrics would now reflect the same.